web/assets/todays-date.js

```diff
@@ -0,0 +1,14 @@
+const todaysDate = () => {
+    const selectTodayLink = document.querySelector('[data-module="select-todays-date"]');
+    const datePicker = document.querySelector('.date-picker');
+
+    if (selectTodayLink !== null && datePicker !== null) {
+        selectTodayLink.addEventListener(
+            "click",
+            function () {
+                datePicker.value = new Date().toJSON().slice(0, 10)
```

Comment:
Should prevent the default action of the click, so it doesn't trigger normal link-like behaviour:
```suggestion
            function (e) {
                e.preventDefault();
                datePicker.value = new Date().toJSON().slice(0, 10)
```